app/controllers/foods_controller.rb

```diff
@@ -1,2 +1,7 @@
 class FoodsController < ApplicationController
+  def destroy
+    @food = Food.find(params[:id])
+    @food.update(deleted: true)
+    redirect_to foods_url
+  end
```

Comment:
food controller should contain create new and show 

as per the requirement 
Should display a list of food added by the logged-in user as in the wireframe (display also quantity of a given food).
Should lead to a form that allows users to add new food.